common/scala/src/main/scala/whisk/common/Logging.scala

```diff
@@ -296,6 +296,8 @@ object LoggingMarkers {
   // Time in invoker
   val INVOKER_ACTIVATION = LogMarkerToken(invoker, activation, start)
   def INVOKER_DOCKER_CMD(cmd: String) = LogMarkerToken(invoker, "docker", start, Some(cmd), Map("cmd" -> cmd))
+  def INVOKER_DOCKER_CMD_TIMEOUT(cmd: String) =
```

Comment:
Add a brief description of this new metric in [metrics.md][1]

[1]: https://github.com/apache/incubator-openwhisk/blob/master/docs/metrics.md

Reply:
Done - thanks for the suggestion. While adding documentation, I also noticed that a different naming scheme for the new counter metric would make more sense. So I also changed from `invoker_dockerCommandTimeout.<command>_count` to `invoker_docker.<command>_timeout`.